marker/util.py

```diff
@@ -80,3 +80,12 @@ def matrix_intersection_area(boxes1: List[List[float]], boxes2: List[List[float]
     height = np.maximum(0, max_y - min_y)
 
     return width * height  # Shape: (N, M)
+
+def rescale_bbox(bbox: List[float], old_size=tuple[float], new_size=tuple[float]):
```

Comment:
You can use PolygonBox.from_bbox(bbox).rescale(old_size, new_size).bbox versus adding a new function.